src/librbd/deep_copy/ObjectCopyRequest.cc

```diff
@@ -91,7 +91,14 @@ void ObjectCopyRequest<I>::send_list_snaps() {
   }
 
   io::SnapIds snap_ids;
-  snap_ids.reserve(1 + m_snap_map.size());
+
+  if(m_src_snap_id_start != 0) {
```

Comment:
nit: white space between 'if' and '(' is missing.

Reply:
Done.